internal/driver/mobile/android.c

```diff
@@ -328,13 +328,37 @@ const char* contentURIGetFileName(uintptr_t jni_env, uintptr_t ctx, char* uriCst
 	return NULL;
 }
 
-bool existsFileURI(char* uriCstr) {
+char *filePath(char *uriCstr) {
 	// Get file path from URI
 	size_t length = strlen(uriCstr)-7;// -7 for 'file://'
 	char* path = malloc(sizeof(char)*(length+1));// +1 for '\0'
 	memcpy(path, &uriCstr[7], length);
 	path[length] = '\0';
 
+    return path;
+}
+
+bool deleteFileURI(char *uriCstr) {
+    char* path = filePath(uriCstr);
```

Comment:
Indentation looks strange here

Reply:
Thanks @Jacalz - seems that GoLand's C indentation isn't totally consistent...